src/frontend/components/TopBar/TopBar.js

```diff
@@ -75,36 +85,54 @@ const TopBar = () => {
     },
   ];
 
+  const history = useHistory();
+  const [dropdownOpen, setDropdownOpen] = useState(false);
+  const { signOut } = useGoogleAuth();
+  const { profilePicture } = useProfilePicture();
   const mapNavDropdowns = () =>
     navDropdowns.map((dropdown) => (
       <NavDropdown key={dropdown.section} dropdown={dropdown} />
     ));
 
+  const onLogout = () => {
+    signOut();
+    history.push('/sign-in');
+  };
+
   return (
-    <AppBar position="sticky">
-      <Toolbar>
-        <NavContainer>
+    <div>
+      <AppBar position="sticky">
+        <Toolbar>
+          <NavContainer>
+            <IconButton
+              style={{ backgroundColor: 'transparent' }}
```

Comment:
You can specify this inside:

`
styled(MUIIconButton)'
    background-color: transparent;
'
`